web/index.html

```diff
@@ -1671,6 +1671,25 @@ <h1 class="modal-title fs-5" id="previewModalLabel">Pipeline Preview</h1>
                     }
                     that.currentPipelineObject['serving'].push(newTask)
                     that.currentPipelineServingTask = that.currentPipelineObject['serving'][that.currentPipelineObject['serving'].length - 1]
+                },
+                exportPipeline() {
+                    var that = this
+                    var pipelineName = this.currentPipelineName
+                    var pipelineObject = this.currentPipelineObject
+                    var pipelineJson = JSON.stringify(pipelineObject, null, 2)
+                    var blob = new Blob([pipelineJson], {
+                        type: "application/json"
+                    });
+
+                    var tempLink = document.createElement("a");
+                    let textArea = document.querySelector("textarea");
+                    var taBlob = new Blob([textArea.value], {type: 'text/plain'});
```

Comment:
what is `taBlob` here?

Reply:
@siliang-j-1225 it is fixed. Thanks for finding the issue.